Launch Portland 2025 CFP (#2241)
writethedocs/www: docs/conf/portland/2025/news/announcing-cfp.rst

We're looking for talk proposals to be delivered in-person, on-stage in Portland, on {{ date.short }}.

You can read our full `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__, and `submit your talk <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__ on the website.

The Call for Proposals will be open until **11:59 PM {{cfp.ends}} PST**, so harness that new year energy!

(If you'd prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for our Atlantic conference held online in a timezone accessible to people all over the world).

Sponsorship Updates
-------------------

Our `sponsorship prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ is updated for {{ year }},
with various improvements based on sponsor feedback.

We're focusing on in-person interaction at the conference, taking full advantage of {{ about.venue }} with a dedicated space for sponsors in the hallway outside the main venue.

Tickets
-------

> Tickets will go on sale in {{ date.tickets_live }}.

Author: We already mentioned this in the opening, so probably worth including again.

```suggestion
Tickets will go on sale in {{ date.tickets_live }}.

Prices are the same as last year:

* **Student or Unemployed Tickets**: *{{tickets.student.price}}*
* **Independent Tickets**: *{{tickets.independent.price}}*
* **Corporate Tickets**: *{{tickets.corporate.price}}*
```

> Tickets will go on sale in {{ date.tickets_live }}.

Reviewer: ```suggestion
Tickets will go on sale in {{ date.tickets_live }}.

Prices are the same as last year:

* **Student or Unemployed Tickets**: *{{tickets.student.price}}*
* **Independent Tickets**: *{{tickets.independent.price}}*
* **Corporate Tickets**: *{{tickets.corporate.price}}*
```

:template: {{year}}/generic.html
:banner: _static/conf/images/headers/portland-2024-speakers.jpg
:og:image: _static/conf/images/headers/portland-2024-speakers.jpg

.. post:: Oct 28, 2024
   :tags: {{shortcode}}-{{year}}, website, cfp, tickets

Announcing the Call for Proposals and Sponsorship Prospectus
============================================================

> We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, opening even earlier than last year, even though the conference back at the start of May this time around.

Author: ```suggestion
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, opening even earlier than last year, even though the conference is back at the start of May this year.
```

Reviewer 2: There are lots of words here that are tripping me up with possible double meanings ( `conference is back at the start...` etc). Also, I'm not sure folks will notice/fully appreciate the relative date shifts (while acknowledging here that it is notable :) ). Does the wider delta give submitters additional time to work on proposals or to polish them once approved?

In any case, just suggestions:

```suggestion
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, which is opening earlier this year than last year, giving you __[more time to blah blah]__ since the conference is in May instead of April this year.
```
or just:
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, which is opening earlier this year!

> We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, opening even earlier than last year, even though the conference back at the start of May this time around.

Reviewer: ```suggestion
We're excited to announce our Portland `Call for Proposals <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/cfp/>`_, opening even earlier than last year.
```

> We're also launching our `Sponsorship Prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ for 2024 with all the information for sponsoring the conference.

Author: ```suggestion
We're also launching our `sponsorship prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ with all the information for sponsoring the conference.
```

Read on for more details about how you can get involved this year.

Call for Proposals
------------------

Speaking at Write the Docs is a fantastic way to share your ideas with our inspiring community of documentarians.

> If there is something you'd really like to see a talk on this year, submit a proposal on it, or refer someone else who would be good!

Reviewer 2: ```suggestion
If there's something you'd really like to see a talk on this year, submit a proposal, or send the submission link to someone else you would like to hear from!
```
(how does "refer" work?)

Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
We're looking for ideas and use-cases from all disciplines and roles, so whether you're a technical writer, developer, UX designer, community manager, or support professional who cares about content and communication, we want to hear from you!

We're looking for talk proposals to be delivered in-person, on-stage in Portland, on {{ date.short }}.

You can read our full `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__, and `submit your talk <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__ on the website.

The Call for Proposals will be open until **11:59 PM {{cfp.ends}} PST**, so harness that new year energy!

(If you'd prefer not to attend an in-person event due to COVID-19 or any other reason, keep an eye out for our Atlantic conference held online in a timezone accessible to people all over the world).

Sponsorship Updates
-------------------

> Our `Sponsorship Prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ is updated for {{ year }},

Author: ```suggestion
Our `sponsorship prospectus <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/sponsors/prospectus/>`_ is updated for {{ year }},
```

with various improvements based on sponsor feedback.

> We're focussing on in-person interaction at the conference, taking full advantage of {{ about.venue }} with plenty of space for sponsors to interact with attendees.

Author: ```suggestion
We're focusing on in-person interaction at the conference, taking full advantage of {{ about.venue }} with a dedicated space for sponsors in the hallway outside the main venue.
```

Reviewer 2: s/focussing/focusing

Tickets
-------

Tickets will go on sale in {{ date.tickets_live }}.

> As always, we keep ticket prices low so that the event is accessible to the widest range of people possible.
> However, if you can't afford the tickets and still wish to attend, we will launch our grants program soon after opening ticket sales.

Author: ```suggestion
As always, we try to keep ticket prices low so that the event is accessible to the widest range of people.
If you can't afford the tickets and still wish to attend, we will launch our grants program soon after opening ticket sales.
```

Write the Docs Atlantic {{ year }}
---------------------------------

If you want to speak at Write the Docs, but can't or don't want to attend an in-person event, keep an eye out for Write the Docs Atlantic, an online conference coming later in {{ year }}.

> This conference will be entirely virtual, in a timezone that works for as many people as possible, so you can present and attend from anywhere.

Author: I might drop this, yea? We don't have anything firm here, so probably not worth mentioning at this stage.

> Write the Docs Atlantic {{ year }}
> ---------------------------------
> 
> If you want to speak at Write the Docs, but can't or don't want to attend an in-person event, keep an eye out for Write the Docs Atlantic, an online conference coming later in {{ year }}.
> 
> This conference will be entirely virtual, in a timezone that works for as many people as possible, so you can present and attend from anywhere.

Reviewer: ```suggestion
```

Stay Updated
------------

Want to find out what's happening with the conference, or enjoy our monthly global community newsletter?

> Sign up to one or more of our `mailing lists <http://eepurl.com/cdWqc5>`_. Your information will never be shared with any third-parties, and you can unsubscribe at any time.

Author: ```suggestion
Sign up to one or more of our `mailing lists <http://eepurl.com/cdWqc5>`_. Your information will never be shared with any third parties, and you can unsubscribe at any time.
```

> Want to connect with other documentarians in real-time? Join our `Slack <https://writethedocs.org/slack/>`_.

Author: ```suggestion
Want to connect with other documentarians in real-time? Join our `Slack <https://www.writethedocs.org/slack/>`_.
```

Thanks
------

> We hope you will join us in-person for yet another Write the Docs conference.

Reviewer 2: ```suggestion
We hope you will join us in person for yet another Write the Docs conference.
```
